gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/editor/GravatarQuickEditorParams.kt

```diff
@@ -8,10 +8,12 @@ import java.util.Objects
  *
  * @property appName The appName of the app that is launching the Quick Editor
  * @property email The email of the user
+ * @property contentLayout The contentLayout of the Quick Editor.
  */
 public class GravatarQuickEditorParams private constructor(
     public val appName: String,
     public val email: Email,
+    public val contentLayout: ContentLayout,
```

Comment:
Yeah, it already seems a bit out of place. If we add it as a parameter to `GravatarQuickEditorBottomSheet` and all, it will be all over the place. Were you thinking of adding some sort of `DisplaySettings`?

Reply:
Yes, the `scope` will be better if we have more options. I think about future potential breaking changes for third parties and this kind of change, switching from a layoutDirection to scope, will be trivial: the build will break, they will notice the layoutDirection is wrapped, will replace it and be done.

I'm not sure sure we should change this now since we don't have plans for other options.

Reply:
>I think about future potential breaking changes for third parties and this kind of change, switching from a layoutDirection to scope, will be trivial: the build will break, they will notice the layoutDirection is wrapped, will replace it and be done.


We shouldn't make breaking changes like that though. Unless bumping up the major version. So it's better to have it defined now in a way that will allow for some modifications. 

I'm leaning toward the suggested solution with the scope, but that's not in line with iOS.

Reply:
It's difficult to guess how the quick-editor will evolve. Maybe we decide to allow users to navigate between different "scopes". In that case, the SDK should allow multiple configurations (one for each possible scope). I don't know if that could make sense 🤷🏽‍♂️. 

Anyway, I would keep it as it is and if at some point we add more scopes, we'll be able to `@Deprecate` the old method until a new major release.

Reply:
Yeah, I agree we can modify it later without breaking changes.